### ISSUE TITLE
[Keras] Determine how to allow parameters of likelihoods to be trainable

### ISSUE THREAD

**Issue by ltiao** [data61/aboleth#61]
With pure TensorFlow, this is easily achieved by defining them as tf.Variables and letting the optimizer take care of the rest. In Keras the loss are specified at model compilation time, where all the trainable parameters must already be specified. May need to resort to a hybrid approach.
